docs/source/api.rst

```diff
@@ -48,3 +59,153 @@ There are two primary endpoints:
 And two secondary endpoints, used to query vulnerability aliases (such as CVEs)
 and vulnerability by CPEs: cpes/ and aliases/
 
+The API is paginated and the default page size is 100. You can change the page size
+by passing the page_size parameter. You can also change the page number by passing
+the page parameter.
+
+.. list-table:: Table for the API endpoints
+   :widths: 30 40 30
+   :header-rows: 1
+
+   * - Endpoint
+     - Query Parameters
+     - Expected Output
+   * - /api/vulnerabilities/
+     -
+       - vulnerability_id (string) = VCID of the vulnerability
+       - page (integer) = page number of the response
+       - page_size (integer) = number of vulnerabilities in each page
+     - Returns a list of vulnerabilities
+   * - /api/vulnerabilities/{id}
+     -
+       - id = internal primary id of the vulnerability
+     - Returns a vulnerability with the given id
+   * - /api/packages
+     -
+       - type (string) = type of the package
```

Comment:
these variables should also be like `type`

Reply:
Done!